Gemfile

```diff
@@ -28,6 +28,7 @@ gem "lograge" # log less so heroku papertrail quits rate limiting our logs
 gem "bootsnap", ">= 1.4.2", require: false # Reduces boot times through caching; required in config/boot.rb
 gem "bugsnag" # tracking errors in prod
 gem "sablon" # Word document templating tool for Case Court Reports
+gem "paranoia", "~> 2.2" # For soft-deleting purpose
```

Comment:
In general, avoid pinning gems unless there is a broken version that we are avoiding. `Gemfile.lock` is enough pinning and a pinned gem often doesn't get upgraded when someone runs `bundle update`

Reply:
@compwron Thank you. I'll notice that